lib/computer_choice.rb

```diff
@@ -0,0 +1,11 @@
+
+def random(selection, name)
+  comp = ["rock", "paper", "scissors"].sample
+  if (selection == "scissors" && comp == "paper") || (selection == "rock" && comp == "scissors") || (selection == "paper" && comp == "rock")
+    return "#{name} wins"
+  elsif (comp == "scissors" && selection == "paper") || (comp == "rock" && selection == "scissors") || (comp == "paper" && selection == "rock")
+    return "#{name} loses"
+  else
+    return "It's a draw"
+  end
```

Comment:
Maybe it's good to return what the computer (your comp variable) has chosen so that users can have more of an insight into the game rather than just getting told who won, for instance: 
"Computer chose paper"
"You chose rock"
"Computer wins!"